docs/guides-getallsecrets.md

```diff
@@ -0,0 +1,39 @@
+# Fetching information from multiple secrets
+
+In some use cases, it might be impractical to bundle all sensistive information into a single secret, or even it is not possible to fully know a given secret name. In such cases, it is possible that an user might need to sync multiple secrets from an external provider into a single Kubernetes Secret. This is possible to be done in external-secrets with the `dataFrom.find` option.
+
+!!! note
+    The secret's contents as defined in the provider are going to be stored in the kubernetes secret as a single key. Currently, it is not possible to apply any decoding Strategy during a find operation.
+
+
+### Fetching secrets matching a given name pattern
+To fetch multiple secrets matching a name pattern from a common SecretStore you can apply the following manifest:
+```yaml
+{% include 'getallsecrets-find-by-name.yaml' %}
+```
+
+Suppose we contain secrets `/path/key1`, `key2/path`, and `path/to/keyring` with their respective values. The above YAML will produce the following kubernetes Secret:
+
+```yaml
+_path_key1: Cg==
+key2_path: Cg==
+path_to_keyring: Cg==
+```
+### Fetching secrets matching a set of metadata tags
+To fetch multiple secrets matching a name pattern from a common SecretStore you can apply the following manifest:
+```yaml
+{% include 'getallsecrets-find-by-tags.yaml' %}
+```
+This will match any secrets containing all of the metadata labels in the `tags` parameter. At least one tag must be provided in order to allow finding secrets by metadata tags.
+
+
+### Searching only in a given path
+Some providers supports filtering out a find operation only to a given path, instead of the root path. In order to use this feature, you can pass `find.path` to filter out these secrets into only this path, instead of the root path.
```

Comment:
```suggestion
Some providers support filtering out a find operation only to a given path, instead of the root path. In order to use this feature, you can pass `find.path` to filter out these secrets into only this path, instead of the root path.
```